bitmap-team-deuces/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const readFile = require(__dirname + '/lib/readwrite').readFile;
+const writeFile = require(__dirname + '/lib/readwrite').writeFile;
+const transform = require(__dirname + '/lib/transform');
+var input = __dirname + '/img/bitmap1.bmp';
+var output = __dirname + '/img/bitmap2.bmp';
+
+function transformBitmap(input, output){
```

Comment:
You may consider exporting this function and specifying input and output as process.argv values so users can call your function on images they specify from the command line (as well as define the output folder).